Source/Core/Core/IOS/USB/Emulated/Skylander.h

```diff
@@ -19,7 +19,36 @@ constexpr u8 MAX_SKYLANDERS = 16;
 
 namespace IOS::HLE::USB
 {
-extern const std::map<const std::pair<const u16, const u16>, const char*> list_skylanders;
+enum class Game
+{
+  SPYROS_ADV,
+  GIANTS,
+  SWAP_FORCE,
+  TRAP_TEAM,
+  SUPERCHARGERS,
+  G_OTHER,
+};
+enum class Element
+{
+  MAGIC,
+  FIRE,
+  AIR,
+  LIFE,
+  UNDEAD,
+  EARTH,
+  WATER,
+  TECH,
+  E_OTHER,
+};
```

Comment:
Will be very annoying to change, but enum class values don't use screaming case (ie: `GIANTS` should be changed to `Giants`)

Reply:
Do you want the names to use camel case or keep the underscores? (ex. SpyrosAdv vs. Spyros_Adv)

Reply:
We typically use CamelCase (no underscore) for enum class.

Reply:
Got it.